src/Leaflet.VectorGrid.js

```diff
@@ -60,6 +63,21 @@ L.VectorGrid = L.GridLayer.extend({
 						}
 					}
 
+					// The user asked to be notified if point type features added to the current vt layer
+					if (pointFeatureHook && feat.type === 1) {
```

Comment:
Do you see any particular reason this should only be done for points?

IMHO, adding this functionality for any feature would be more powerful and avoid cluttering the API with type specific hooks.

If added as `oneachfeature`, we could pass it the created layer as well as the original feature, more or less exactly as `L.GeoJSON` does. Also, I imagine doing it this way would heavily reduce the amount of code needed.